rma_product_exchange/models/rma_order_line.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Akretion
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from odoo import fields, models
+
+
+class RmaOrderLine(models.Model):
+    _inherit = "rma.order.line"
+
+    new_product_id = fields.Many2one(
```

Comment:
Maybe a help here to explain what will happen if the field is set ?
Explain that will impact the shipping

Reply:
@florian-dacosta , it's done.